west.yml

```diff
@@ -109,7 +109,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: ddf36f735bc44bed5877373d1c044372ea405cf9
+      revision: pull/641/head
```

Comment:
`19377140adb905e361614980882cdad537a65b14`

Reply:
just updated to latest: 19377140adb905e361614980882cdad537a65b14